operations/observability/mixins/cross-teams/dashboards/gitpod-grpc-client.json

```diff
@@ -768,6 +768,37 @@
         "skipUrlSync": false,
         "sort": 5,
         "type": "query"
+      },
+      {
```

Comment:
I think this should have the All value set to `"allValue": ".*",`, such that the regex matches all.